textmap.py

```diff
@@ -130,6 +130,8 @@ def placeText(
             dy = myGlyph.dy
             shift_x = myGlyph.shift_x
             shift_y = myGlyph.shift_x
+            glyph_offset_x = myGlyph.tile_index * width # for type BuiltinFont, this creates the x-offset in the glyph bitmap.
```

Comment:
In contrast to BDF-imported fonts, the BuiltinFont glyph consists of the full bitmap of all glyphs and a tile_index which indicates which glyph should be used.  We can use the tile_index and glyph width (the same width for all glyphs in a fixed size font) to reference into the glyph bitmap when we copy it to the main bitmap.